pypeit/core/extract.py

```diff
@@ -1510,22 +1481,22 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
 
     ## Okay now loop over all the regular aps and exclude any which within the fwhm of the hand_extract_APERTURES
     if nobj_reg > 0 and hand_extract_dict is not None:
-        spat_pixpos = sobjs.spat_pixpos
+        spat_pixpos = sobjs.SPAT_PIXPOS
         hand_flag = sobjs.hand_extract_flag
-        spec_fwhm = sobjs.fwhm
-        #spat_pixpos = np.array([spec.spat_pixpos for spec in specobjs])
+        spec_fwhm = sobjs.FWHM
+        #spat_pixpos = np.array([spec.SPAT_PIXPOS for spec in specobjs])
         #hand_flag = np.array([spec.hand_extract_flag for spec in specobjs])
-        #spec_fwhm = np.array([spec.fwhm for spec in specobjs])
+        #spec_fwhm = np.array([spec.FWHM for spec in specobjs])
         reg_ind, = np.where(~hand_flag)
```

Comment:
Can you please change this ~ to an np.invert. I realize it was not introduced by you, but I just caught it.